docs/source/examples/spot-jobs.rst

```diff
@@ -175,3 +202,17 @@ Here are some commands for managed spot jobs. Check :code:`sky spot --help` for
     # Cancel a spot job by name
     $ sky spot cancel -n bert-qa
 
+
+Advanced note (spot controller)
```

Comment:
Should we do the other way around?:
Spot Controller (advanced note)

Reply:
Good point! Changed. Thanks!